drivers/telescope/lx200_10micron.cpp

```diff
@@ -1184,9 +1150,9 @@ bool LX200_10MICRON::ISNewSwitch(const char *dev, const char *name, ISState *sta
             IDSetSwitch(&AlignmentSP, nullptr);
             return true;
         }
+        if (strcmp(TrackSatSP.name, name) == 0)
+          {
```

Comment:
spacing

Reply:
?

Reply:
the { should move 2 positions to the left

Reply:
Right, we use astyle for the formatting. It's documented in the README.md in the Github page.

Reply:
I formatted the entire file using astyle and the config stated in the README. Let me know if something was done incorrectly